config-model/src/test/java/com/yahoo/vespa/model/container/xml/ContainerModelBuilderTest.java

```diff
@@ -67,6 +67,31 @@
  */
 public class ContainerModelBuilderTest extends ContainerModelBuilderTestBase {
 
+    @Test
+    public void verify_jvm_tag_with_attributes() throws IOException, SAXException {
+        String servicesXml =
+                "<jdisc version='1.0'>" +
```

Comment:
I prefer _container_ to _jdisc_ wherever, as the _jdisc_ name will go away some time ...

Reply:
Unified on jdisc => container